pydtk/models/rosbag2.py

```diff
@@ -0,0 +1,365 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright Toolkit Authors
+
+import logging
+
+from abc import ABC
+
+import cv2
+from flatdict import FlatDict
+import numpy as np
+from pandas import DataFrame
+import re
+import yaml
```

Comment:
使われていないパッケージがあります
```suggestion
import numpy as np
from pandas import DataFrame
```

Reply:
fixed but why flake8 miss these errors...?

Reply:
Probably because `F401: 'package' imported but unused` is ignored by tox.ini
https://github.com/dataware-tools/pydtk/blob/feat/support-ros2/tox.ini#L33